app/_how-tos/configure-konnect-identity-oidc.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Configure the OIDC plugin with {{site.konnect_short_name}} Identity
+content_type: how_to
+breadcrumbs:
+  - /konnect-identity/
+
+tech_preview: true
+entities:
+  - route
+  - service
+  - plugin
+plugins:
+  - openid-connect
+products:
+    - gateway
+works_on:
+  - konnect
+tags:
+    - authentication
+description: Learn how to configure {{site.konnect_short_name}} Identity with OIDC.
+
+tldr: 
+  q: How do I configure {{site.konnect_short_name}} Identity with OIDC?
+  a: | 
+    Create a {{site.konnect_short_name}} Identity auth server, scope, claim, and client. Configure the OIDC plugin with your issuer URL, client ID, and client secret. Generate a client token by sending a GET request to `$ISSUER_URL/oauth/token` and use the access token in a header when you send a request to the Gateway Service.
+
+tools:
+    # - konnect-api
+    - deck
+  
+prereqs:
+  inline: 
+    - title: "{{site.konnect_short_name}} Labs"
```

Comment:
Since we are going public beta, Kong Identity will be removed from Labs. The API FF will be enabled for all customers. We can remove this pre-req.